README.adoc

```diff
@@ -146,6 +146,8 @@ include::finish/src/main/liberty/config/userRegistry.xml[tags=**]
 
 The registry has four users, `bob`, `alice`, `carl` and `dave`, and four groups, `Employee`, `TeamLead`, `Manager` and `PartTime`. Each user belongs to one or more groups.
 
+It is not recommended to store passwords in plain text. For simplicity reasons, the application in this guide uses XOR encoding to encode the passwords, as you can see in the `userRegistry.xml`. This passwords are encoded using the Liberty `securityUtility` command.
```

Comment:
"This passwords are encoded" => Should be "These passwords are encoded" or "The passwords are encoded"?

Reply:
I will change this to "The passwords are encoded"